eng/Versions.props

```diff
@@ -3,18 +3,26 @@
   <PropertyGroup>
     <!-- This repo version -->
     <VersionPrefix>3.0.0</VersionPrefix>
+    <!-- Always use shipping version instead of dummy versions -->
+    <DotNetUseShippingVersions>true</DotNetUseShippingVersions>
```

Comment:
This was added to some command lines in the script files. I would remove it there if we are setting it here.